config/environments/test.rb

```diff
@@ -37,7 +37,7 @@
   config.i18n.default_locale = "en"
 
   # Tests assume Aus timezone.
-  config.time_zone = "Melbourne"
+  config.time_zone = "Australia/Melbourne"
```

Comment:
Could you please squash this commit into the previous? Might as well just have one commit with the correct timezone.